inc/utils/namespace.php

```diff
@@ -871,3 +871,13 @@ function get_countries() : array {
 		'ZW' => 'Zimbabwe',
 	];
 }
+
+/**
+ * Get a letter of the alphabet corresponding to the passed zero based index.
+ *
+ * @param integer $index Letter of the alphabet to get.
+ * @return string
+ */
+function get_letter( int $index ) : string {
+	return strtoupper( 'abcdefghijklmnopqrstuvwxyz'[ $index ] ?? '' );
```

Comment:
#morningfun #beforecoffee 

```php
function get_letter( int $index ) : string {
    $letters = $index / 26;
    return str_repeat( 'z', floor( $letters ) ) . ( fmod( $letters, 1 ) ? chr( ord( 'a' ) + 26 * fmod( $letters, 1 ) - 1 ) : '' );
}

var_dump( implode( ",", array_map( 'get_letter', range(1, 100) ) ) );

// "a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z,za,zb,zc,zd,ze,zf,zg,zh,zi,zj,zk,zl,zm,zn,zo,zp,zq,zr,zs,zt,zu,zv,zw,zx,zy,zz,zza,zzb,zzc,zzd,zze,zzf,zzg,zzh,zzi,zzj,zzk,zzl,zzm,zzn,zzo,zzp,zzq,zzr,zzs,zzt,zzu,zzv,zzw,zzx,zzy,zzz,zzza,zzzb,zzzc,zzzd,zzze,zzzf,zzzg,zzzh,zzzi,zzzj,zzzk,zzzl,zzzm,zzzn,zzzo,zzzp,zzzq,zzzr,zzzs,zzzt,zzzu,zzzv"
```

Woke up as I started to optimize for `zaa, zab, ...` 😄 (will do so later 😛)

Reply:
oh man, not done any code katas in ages 😂 

This a zero based index so 0 should equal "a". I'd probably go for a-z then aa, ab, ac, ... ba, bb, bc etc... Might see if I can figure that out.

Reply:
Well, definitely not needed here :D i just did some work with `ord` and `chr` before so thought I'd try if it fits.